code/go/0chain.net/blobbercore/handler/handler.go

```diff
@@ -240,6 +241,15 @@ func RefsHandler(ctx context.Context, r *http.Request) (interface{}, error) {
 	return response, nil
 }
 
+func RecentRefsRequestHandler(ctx context.Context, r *http.Request) (interface{}, error) {
+	ctx = setupHandlerContext(ctx, r)
+	response, err := storageHandler.GetRecentlyAddedRefs(ctx, r)
+	if err != nil {
+		return nil, err
+	}
+	return response, nil
```

Comment:
It does one thing though. If handler would have returned both response and error then it will return both response and error.
It can be decided in this function whether to return nil response if error occurs.